src/i18n/MControlCenter_pt.ts

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_PT" sourcelanguage="en_US">
+<context>
+    <name>MainWindow</name>
+    <message>
+        <source>Info</source>
+        <translation type=Informação></translation>
```

Comment:
The translated strings should look like this
```suggestion
        <translation>Informação</translation>
```